app/views/admin_mailer/notify_moderators_of_comment_spam.html.erb

```diff
@@ -17,7 +17,7 @@
 <hr>
 <div class="footer">
 
-  <p>Visit the spam page here: <a href="http://publiclab.org/spam">http://publiclab.org/spam</a></p>
+  <p>Visit the spam page here: <a href="https://publiclab.org/spam">https://publiclab.org/spam</a></p>
```

Comment:
Great catch! Thanks!

Actually i'll suggest something even more - see how we can fetch the hostname and insert it, in case this codebase is ever run on a different domain name? 

From line 22:

```suggestion
  <p>Visit the spam page here: <a href="https://<%= ActionMailer::Base.default_url_options[:host] %>/spam">https://<%= ActionMailer::Base.default_url_options[:host] %>/spam</a></p>
```

It's a little harder to read, but what do you think?

Reply:
Yes, it is a bit harder to read. But if it is okay, I am happy to give it a try.